### PR TITLE
[new release] ocaml-lsp-server, lsp and jsonrpc (1.4.1)

### PR DESCRIPTION
**Pull request by rgrinberg**
LSP Server for OCaml

- Project page: <a href="https://github.com/ocaml/ocaml-lsp">https://github.com/ocaml/ocaml-lsp</a>

## Fixes

- Backport fixes from merlin (ocaml/ocaml-lsp#382, ocaml/ocaml-lsp#383)

- Encode request & notification `params` in a list. This is required by the
  spec. (ocaml/ocaml-lsp#351)

### CODE REVIEW

File: packages/ocaml-lsp-server/ocaml-lsp-server.1.4.1/opam [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  "stdlib-shims"
+  "ppx_yojson_conv_lib"
+  "dune-build-info"
+  "dot-merlin-reader"
+  "csexp" {>= "1.2.3"}
+  "result" {>= "1.5"}
+  "ocamlformat" {with-test}
+  "ocamlfind" {>= "1.5.2"}
+  "odoc" {with-doc}
+  "ocaml" {>= "4.06" & < "4.12"}
```
Comment (mseri) [inline packages/ocaml-lsp-server/ocaml-lsp-server.1.4.1/opam:29]: Just double checking, this release does not add support for 4.12

Comment (avsm) [reply packages/ocaml-lsp-server/ocaml-lsp-server.1.4.1/opam:29]: That's right -- @rgrinberg indicated that he's working on 4.12 support for the next release of lsp-server.

Comment (mseri) [reply packages/ocaml-lsp-server/ocaml-lsp-server.1.4.1/opam:29]: Does this need extra testing on bsd/osx before merging or is good to go?

Comment (kit-ty-kate) [reply packages/ocaml-lsp-server/ocaml-lsp-server.1.4.1/opam:29]: vscode isn't available on BSDs but I just tested on macOS just in case, it's all good.

**Comment by kit-ty-kate** [comment]
thanks!

**Pull request merged**